ecs-cli/modules/cli/local/options/options.go

```diff
@@ -54,3 +56,21 @@ func ValidateFlagPairs(c *cli.Context) error {
 	}
 	return nil
 }
+
+// ContainerSearchInfo show the task definition filepath or arn/family message on local down.
+func ContainerSearchInfo(c *cli.Context) {
+	if c.IsSet(flags.TaskDefinitionFile) {
+		logrus.Infof("Searching for containers matching --%s=%s", flags.TaskDefinitionFile, c.String(flags.TaskDefinitionFile))
```

Comment:
I'm not sure we need to include the flag dashes here. It might be friendlier to have something like  `"Searching for containers run from local file %s, c.String(flags.TaskDefinitionFile))` and similarly, "Searching for containers run from remote task definition ...." for one pulled from a remote task def.

Reply:
Done!